fal_project.yml

```diff
@@ -0,0 +1,10 @@
+environments:
```

Comment:
wondering if this makes it too complicated for the most basic example.

Reply:
Well, I thought it's better than saying "do all these pip installs"

Reply:
haha so true.